### ISSUE TITLE
[bug][1.1.0] no types in the npm package

### ISSUE THREAD

**Issue by xr0master** [y-takey/chartjs-plugin-stacked100#54]
the `index.d.ts` file is not in the build folder

![image](https://user-images.githubusercontent.com/8962340/144425984-25f9395f-337f-4e2f-877e-ef44b69646a1.png)
![image](https://user-images.githubusercontent.com/8962340/144426019-fed47d2b-0bf9-430b-a199-d3cc1d625f95.png)

**Comment by xr0master** [comment]
nvm, the version 1.0.3 with types anyway doesn't support Chartjs v3 unfortunately.

This code will help the plugin support Chartjs v3 types.

```ts 
declare module 'chart.js' {
    // eslint-disable-next-line @typescript-eslint/no-unused-vars
    export interface PluginOptionsByType<TType extends ChartType> {
        stacked100: {
              enable: boolean;
              replaceTooltipLabel?: boolean;
              fixNegativeScale?: boolean;
              precision?: number;
         };
    }
}
```

**Comment by y-takey** [comment]
Hi @xr0master , thank you very much!
I fiexed this issue and published v1.1.1 :)

**Comment by xr0master** [comment]
It works great! Thank you.
